index.js

```diff
@@ -93,6 +103,8 @@ module.exports = {
             item_doesnt_exist: 'Thành phần không tồn tại',
             http_error: 'Lỗi giao tiếp với máy chủ',
             canceled: 'Hủy bỏ hành động',
+            data_provider_error: 'lỗi dataProvider: Kiểm tra bàn điều khiển để biết chi tiết.',
```

Comment:
I think 'dataProvider error. Check the console for details.' will be 'Lỗi dataProvider. Kiểm tra console để biết chi tiết.' We can not translate for console word it will not make sense in this case.